psalm.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0"?>
 <psalm
-    totallyTyped="false"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xmlns="https://getpsalm.org/schema/config"
-    xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"
+    xsi:schemaLocation="https://getpsalm.org/schema/config ../Libraries/vimeo/psalm/config.xsd"
     errorBaseline="Packages/Framework/psalm-baseline.xml"
+    resolveFromConfigFile="false"
```

Comment:
Without this, the paths were not resolved from "here", even though it's supposed to be the default. 🤷‍♂️